src/screens/konnectors/LauncherView.js

```diff
@@ -201,9 +201,7 @@ class LauncherView extends Component {
             </View>
           </>
         ) : (
-          <View>
-            <Text>Loading...</Text>
-          </View>
+          <View />
```

Comment:
can't we just return null instead of a view?

Reply:
It makes the most sense I agree, I'm updating it